buku

```diff
@@ -2456,17 +2458,19 @@ class BukuDb:
             True on success, False on failure.
         """
 
-        ff_bm_db_path = None
+        ff_bm_db_paths = {}
+        firefox_profile = firefox_profile and [firefox_profile]
 
         if sys.platform.startswith(('linux', 'freebsd', 'openbsd')):
             gc_bm_db_path = '~/.config/google-chrome/Default/Bookmarks'
             cb_bm_db_path = '~/.config/chromium/Default/Bookmarks'
             vi_bm_db_path = '~/.config/vivaldi/Default/Bookmarks'
 
             default_ff_folder = os.path.expanduser('~/.mozilla/firefox')
-            profile = get_firefox_profile_name(default_ff_folder)
-            if profile:
-                ff_bm_db_path = '~/.mozilla/firefox/{}/places.sqlite'.format(profile)
+            profiles = firefox_profile or get_firefox_profile_names(default_ff_folder)
+            if profiles:
+                ff_bm_db_paths = {s: '~/.mozilla/firefox/{}/places.sqlite'.format(s)
+                                  for s in profiles}
```

Comment:
Keys are printed in prompt (when more than 1 profiles were found).

Dicts retain insertion order (since Python 3.7).